examples/chart/teleport-cluster/templates/service.yaml

```
@@ -17,7 +17,11 @@ metadata:
     {{- end }}
   {{- end }}
 spec:
-  type: LoadBalancer
+  {{- $type := default "LoadBalancer" .Values.service.type }}
+  type: {{- $type }}
```

This could be simplified a bit (as `$type` isn't needed or used anywhere else in the template)
```suggestion
  type: {{- default "LoadBalancer" .Values.service.type }}
```